node/lib/util/status_util.js

```diff
@@ -663,4 +670,12 @@ Before proceeding, you must complete the cherry-pick in progress (by running
 Please resolve outstanding conflicts before proceeding:
 ${PrintStatusUtil.printRepoStatus(status, "")}`);
     }
+    if (null !== status.sequencerState) {
+        const command =
+               PrintStatusUtil.getSequencerCommand(status.sequencerState.type);
+        throw new UserError(`\
+Before proceeding, you must complete the ${command} in progress (by running
```

Comment:
This is a lot of repeated text from above.  Is there any way to unify this text with the above text?

Reply:
When this refactoring is over that will all be gone.

Reply:
As merge, rebase, and cherry-pick will be redone in terms of the sequencer. The next few PRs after this one will contain a lot of deleted code!